README.md

```diff
@@ -63,6 +63,20 @@ Here the list of the supported platforms:
 
 We're looking forward to more platforms to support in the future. Contributions are more than welcome.
 
+## Examples
+
+You can find some **examples** in this repository to help you set up your generator environment.
+
+* [sample-generated-code](/sample-generated-code) Contains an example of an Android Library where the code is generated inside the `/scr/main/java` folder. You can use this example to see how the generated code will _look like_ (like [here](/sample-generated-code/src/min/java/com/yelp/codegen/generatecodesamples/apis/ResourceApi.kt))
+
+* [sample-groovy-android](/sample-groovy-android) Contains an example of an Android Library configured with a `build.gradle` file, using Groovy as scripting language.
```

Comment:
nit. I would not mention the following ", using Groovy as scripting language"

Reply:
I've rephrased a bit